lib/auth.js

```diff
@@ -59,7 +59,7 @@ SwaggerAuthorizations.prototype.apply = function (obj, securities) {
 
   _.each(this.authz, function (auth, authName) {
     if(applyAll || _.includes(flattenedSecurities, authName)) {
-      var newStatus = auth.apply(obj);
+      var newStatus = (typeof auth.apply !== 'undefined') ? auth.apply(obj) : true; // How should we handle a security that does not implement a SwaggerAuthorization?
```

Comment:
I assume maybe ```typeof auth.apply === 'function'```` might be more appropriate. Should a default case here be true or false if the passed in security is not a 'SwaggerAuthorization' object